docs/zkapps/tutorials/09-recursion.mdx

```diff
@@ -266,7 +266,7 @@ class RollupState extends Struct({
 
 const Rollup = ZkProgram({
   name: "rollup-example",
-  publicInputType: Field,
+  publicInput: Field,
```

Comment:
this fix is outside the scope of this PR

can you remove this file update from the PR?
One way to get this file from the main branch is:

`git checkout main docs/zkapps/o1js/recursion.mdx`